app/assets/javascripts/discourse/components/groups-form-email-fields.js.es6

```diff
@@ -0,0 +1 @@
+export default Ember.Component.extend({});
```

Comment:
Can you import `@ember/component` please? It's our new standard way of doing this.

Also super minor but you can `Component.extend()` without the empty object.